cgo/types.go

```diff
@@ -617,6 +617,6 @@ func (r FvmMachineExecuteResponse) copy() FvmMachineExecuteResponseGo {
 		MinerTipHi:  uint64(r.miner_tip_hi),
 		MinerTipLo:  uint64(r.miner_tip_lo),
 		ExecTrace:   r.exec_trace.copy(),
-		FailureInfo: string(r.failure_info.copy()),
+		FailureInfo: string(r.failure_info.slice()),
```

Comment:
are you sure that `string()` copies?

Reply:
It's supposed to? I think go will only optimize this in _very_ special cases. Basically, where it can inline things.

Reply:
I'm 99% sure it doesn't.

Reply:
then this needs to be reverted

Reply:
I'm 99.99999999999% sure it gets copied.

Reply:
Ok, then it is the 1%, I thought it was ‘string(string)’ cast.